pages/common/git-rev-list.md

```diff
@@ -0,0 +1,16 @@
+# git rev-list
+
+> Lists commits in reverse chronological order.
+> More information: <https://git-scm.com/docs/git-rev-list>.
+
+- List all commits on current branch:
+
+`git rev-list {{HEAD}}`
```

Comment:
`HEAD` is assumed by default, right? I'd simply remove the token from the command.

Reply:
Unfortunately there's no default :/

Reply:
Thanks for confirming.